Formula/shadowenv.rb

```diff
@@ -0,0 +1,43 @@
+class Shadowenv < Formula
+  desc "Reversible directory-local environment variable manipulations"
+  homepage "https://shopify.github.io/shadowenv/"
+  url "https://github.com/Shopify/shadowenv/archive/1.0.0.tar.gz"
+  sha256 "8d00ec4db578c21c7219754595f8ac5f40ba5a2a1107b4ddb2dbf44da5b44804"
+
+  depends_on "rustup-init" => :build
+
+  def install
+    system "rustup-init", "-y"
+    ENV["PATH"] = "#{ENV["CARGO_HOME"]}/bin:#{ENV["PATH"]}"
+    system "rustup", "toolchain", "install", "nightly-2018-12-26"
```

Comment:
What is `+nightly-2018-12-26`?

Reply:
Rust has a weird system of features only available in nightlies for long periods of time.  The 1.0.1 version of shadowenv has been updated to use a dependency no longer relying on this, so this can go away.